main.py

```diff
@@ -340,6 +340,7 @@ def enable_interface(self, interface_enable_id=None, dpid=None):
         if not error_list:
             log.info(f"Storing administrative state for enabled interfaces.")
             self.save_status_on_storehouse()
+            self.notify_switch_enabled(dpid)  # The switch is also enabled
```

Comment:
Should an interface_enable really trigger a switch_enabled  event?

Reply:
Today, when an interface connected to a disabled switch is enabled, the switch is also enabled, but this activation does not trigger the `switch_enabled` event. Therefore, it is necessary to trigger this event here.

Reply:
After talking with @beraldoleal we decided that this behavior is wrong. The right thing to do is to raise an error if the user tries to enable an interface in a disabled switch.

Reply:
An issue was created to change this behavior https://github.com/kytos/kytos/issues/1233